homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -450,6 +459,15 @@ async def async_remote_control_move_step(
             duration=duration,
         )
 
+    async def async_goto(self, x_coord: int = 25500, y_coord: int = 25500):
```

Comment:
it is not customary to define defualt values for service calls.

Reply:
Removed the default values